app/components/classifications-ribbon.cjsx

```diff
@@ -1,5 +1,7 @@
 React = require 'react'
 apiClient = require 'panoptes-client/lib/api-client'
+getColorFromString = require('../lib/get-color-from-string').default
```

Comment:
I think you can use backticked javascript to load the js files eg. `import getColorFromString from '../lib/get-color-from-string';`